test/testapps/BasicTestApp/BasicTestApp.csproj

```diff
@@ -6,14 +6,50 @@
     <!-- Local alternative to <RunArguments>blazor serve</RunArguments> -->
     <RunCommand>dotnet</RunCommand>
     <RunArguments>run --project ../../../src/Microsoft.AspNetCore.Blazor.Cli serve --pathbase /subdir</RunArguments>
+    <TypeScriptToolsVersion>2.8</TypeScriptToolsVersion>
   </PropertyGroup>
 
   <!-- Local alternative to <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" /> -->
   <Import Project="..\..\..\src\Microsoft.AspNetCore.Blazor.Build\ReferenceFromSource.props" />
+  <ItemGroup>
+    <BlazorGenerate Remove="InteropTest\InteropComponent.cshtml" />
+    <BlazorGenerate Remove="InteropTest\TreeNode.cshtml" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Remove="wwwroot\js\interopTests.ts" />
+  </ItemGroup>
```

Comment:
Guessing the changes in this block aren't intentional

Reply:
Yeah, this needs cleanup